cyral/internal/sidecar/listener/datasource_test.go

```diff
@@ -57,14 +57,14 @@ func TestAccSidecarListenerDataSource(t *testing.T) {
 				Config: testConfigTypeFilter,
 				Check:  testFuncTypeFilter,
 			},
-			{
-				Config: testConfigPortFilter,
-				Check:  testFuncPortFilter,
-			},
-			{
-				Config: testConfigTypePortFilter,
-				Check:  testFuncTypePortFilter,
-			},
+			// {
+			// 	Config: testConfigPortFilter,
+			// 	Check:  testFuncPortFilter,
+			// },
+			// {
+			// 	Config: testConfigTypePortFilter,
+			// 	Check:  testFuncTypePortFilter,
+			// },
```

Comment:
Why this is being commented? If this test case does not apply anymore we can remove it instead, otherwise we should uncomment it.

Reply:
I commented as part of some debugging sessions and forgot to reinstate the code. Thanks! Addressed in 1d4545f311c3daf857fc7a7ab532fac26aa70c62.